cloud/jenkins/psmdb_operator_eks_latest.groovy

```diff
@@ -309,12 +350,12 @@ pipeline {
             description: 'percona-server-mongodb-operator repository',
             name: 'GIT_REPO')
```

Comment:
Could we add to params test_list, test_suite and ignore_previous_run? How we have for ps for example?